app/javascript/app/index.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import Routes from './Routes';
-import store from '../../../redux';
+import Routes from './config/Routes';
+import store from '../redux';
 
 // todo: convert this to use ManageIQ.redux.addReducer, ManageIQ.redux.store
 // <Provider store={store}>{componentRegistry.markup(component, data, store)}</Provider>,
```

Comment:
`app/javascript/app...` is a little repetitive.

I know it follows the convention [here](https://gist.github.com/ryanflorence/daafb1e3cb8ad740b346#shared-modules) -- but just checking if we should consider a different folder name instead of `app`

Reply:
hey @AparnaKarve - do you have another suggestion? I am happy to change it quickly today... I was on the fence myself (knowing that `app` is Rails plugin app and also the React front end app). I think this was more to prepare for long term efforts.

Reply:
i think per our conversation, we can go with "react" instead. I am updating this now...

Reply:
updated... can you confirm @AparnaKarve ? If it's good... please merge. ;)